### PR TITLE
Update FakeIP addresses and Ports

### LINKED ISSUE #119: Update FakeIP addresses and Ports
IP addresses and Ports to be updated according to [FakeIpList](https://github.com/openBackhaul/ApplicationPattern/blob/develop/doc/TestingApplications/Infrastructure/SdnLaboratory/FakeAddresses/IpAddresses.md):
- ServiceList
- CONFIGfile
- OAS

**IP_1 and TCP_1 to be applied.**
**Release numbers to be regarded!**

### PR DESCRIPTION
**Pull request by kmohr-soprasteria**
Fixes #119

### CODE REVIEW

**Review by IswaryaaS — CHANGES_REQUESTED** [review]
In config.json file, the client entries for TypeApprovalRegister, OperationKeyManagement and CurrentController are missing in logical-termination-point list. We could also see this being not present in config file of version 1.0.0 as well. Though absence of these entries does not have any significant affect, just to maintain harmony between service.yaml file and config.yaml file, please let me know if we can add the entries here or can we raise a separate issue for adding the entries.
@openBackhaul , kindly requesting for your thoughts

**Review by IswaryaaS — COMMENTED** [review]
I observed few scenarios which is not related to this issue, please find them below. 
1. /v1/regard-application and /v1/document-approval-status - the pattern of release-number needs an update to "^([0-9]{1,2})\.([0-9]{1,2})\.([0-9]{1,2})$"
2. Since, the attribute name of "application-release-number" is updated to "release-number" in  similar services, can we also update them in /v1/regard-application and /v1/document-approval-status
Kindly let me know your views

**Comment by openBackhaul** [comment]
Added HttpClients and TcpClients for TypeApprovalRegister, OperationKeyManagement and CurrentController in CONFIGfile as proposed.

Changed attribute name from _application-release-number_ to _release-number_ at _/v1/regard-application_ and _/v1/document-approval-status_ as proposed.

Changed attribute name from _application-release-number_ to _release-number_ at _/v1/list-applications_, too.

Changed patterns for _release-number_ at _/v1/regard-application_ and _/v1/document-approval-status_ as proposed.

File: TypeApprovalRegister.yaml [hunk 1]
```diff
@@ -209,8 +209,8 @@ paths:
                         subscriber-protocol: 'HTTP'
                         subscriber-address:
                           ip-address:
-                            ipv-4-address: '1.1.3.14'
-                        subscriber-port: 3014
+                            ipv-4-address: '1.1.3.7'
+                        subscriber-port: 3007
               responses:
                 '204':
                   description: 'Subscribed for notifications about approval status updates'
```
Comment (IswaryaaS) [inline TypeApprovalRegister.yaml:216]: As per the latest decision, to remove api-segment from http-s/c and tcp-s/c, the uuid's are updated in config and service list files. Similarly can we also update in OAS as well in this PR or shall we create a separate issue for updating the uuid(s) in OAS. Please let me know your views.

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

File: TypeApprovalRegister.yaml (hunk 1 above)
Comment (IswaryaaS) [reply TypeApprovalRegister.yaml:216]: A separate issue has been already created for this purpose. hence resolving the conversation.

File: TypeApprovalRegister+config.json [hunk 2]
```diff
@@ -1372,10 +1372,57 @@
...
                     },
-                    "remote-port": 1000
+                    "remote-port": 3008
+                  }
+                }
+              }
+            ]
+          },
+          {
+            "uuid": "tar-2-0-0-http-c-tar-1-0-0-000",
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply TypeApprovalRegister+config.json:1385]: Changed as proposed.

File: TypeApprovalRegister+config.json [hunk 3]
```diff
@@ -1372,10 +1372,57 @@
...
+                  "http-client-interface-configuration": {
+                    "application-name": "TypeApprovalRegister",
+                    "release-number": "1.0.0"
+                  }
+                }
+              }
+            ]
+          },
+          {
+            "uuid": "tar-2-0-0-tcp-c-tar-1-0-0-000",
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply TypeApprovalRegister+config.json:1406]: Changed as proposed.

File: TypeApprovalRegister+config.json [hunk 4]
```diff
@@ -1787,10 +1834,104 @@
...
+              }
+            ]
+          },
+          {
+            "uuid": "tar-2-0-0-http-c-cc-1-0-0-000",
+            "ltp-direction": "core-model-1-4:TERMINATION_DIRECTION_SINK",
+            "client-ltp": [
+            ],
+            "server-ltp": [
+              "tar-2-0-0-tco-c-cc-1-0-0-000"
```
[the remark this answers is not in the document]
Comment (openBackhaul) [reply TypeApprovalRegister+config.json:1915]: Corrected as proposed.

**Comment by openBackhaul** [comment]
Many thanks for reviewing.

**Review by IswaryaaS — APPROVED** [review]

**Pull request merged**